allvers/2007-07-01-GPL-2.0-from-gnu.org.html.notes

```diff
@@ -0,0 +1,10 @@
+
+Add the HTML variant of v6 #1
+
+From https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+In this version (beyond the HTML formatting) the differences are:
+   angle brackets and "yyyy" instead of "year" in the appendix
+
+Fix #1
+
+https://web.archive.org/web/20070701213050/https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
This is identical to http://web.archive.org/web/20050609004347/http://www.gnu.org/licenses/gpl.html except for HTML formatting and it wasn't introduced in 2007.

Reply:
Fixed in cebf5e5d739ef187df3cfc5df6e26b63784f4eba